src/map.cpp

```diff
@@ -4711,6 +4712,16 @@ item &map::add_item_at( const tripoint &p,
     if( new_item.has_flag("ACT_IN_FIRE") && get_field( p, fd_fire ) != nullptr ) {
         new_item.active = true;
     }
+    
```

Comment:
`monster::drop_items_on_death()`

`put_items_from_loc` returns a vector of pointers to created items, so you can filthify them there.